### PR TITLE
A couple of clarifications for the tutorial

### PR DESCRIPTION
**Pull request by thams**

### CODE REVIEW

File: nameservice/Makefile [hunk 1]
```diff
@@ -15,12 +15,12 @@ include Makefile.ledger
 all: install
 
 install: go.sum
-		go install -mod=readonly $(BUILD_FLAGS) ./cmd/nsd
+		go install  $(BUILD_FLAGS) ./cmd/nsd
```
Comment (marbar3778) [inline nameservice/Makefile]: what is the reason to remove -mod=readonly? i do believe this updates some deps and this is not wanted.

**Comment by hschoenburg** [comment]
@thams  can you make sure to check the "Allow edits from maintainers" check box on your PR? Then I can get your PR merged promptly. Thank you for the contribution.

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by thams** [comment]
Allow edits is now checked

File: nameservice/Makefile (hunk 1 above)
Comment (thams) [reply nameservice/Makefile]: Was needed for mine to build. Can’t remember why now.

Comment (marbar3778) [reply nameservice/Makefile]: can you undo this change, otherwise you will be pulling new versions which are not supported.

Comment (thams) [reply nameservice/Makefile]: Done and pushed.

Comment (marbar3778) [reply nameservice/Makefile]: 😄 thanks

**Pull request merged**